static/includes/Reference/ServicesTFTPFields.md.part

```diff
@@ -8,7 +8,7 @@
 
 | Name | Description |
 |------|-------------|
-| **Host** | The default host to use for TFTP transfers. Enter an IP address. For example, *192.0.2.1* or in **Shell** `192.0.2.1` |
+| **Host** | The default host to use for TFTP transfers. The host IP address displays in the dropdown list of options. Or in **Shell**, enter an IP address. For example, `192.0.2.1`. |
```

Comment:
When talking about a variable even in CLI use italics except in a full command string where we are using { } around the variable and then calling it out as a variable after the command string. Suggest this change:
```suggestion
| **Host** | The default host to use for TFTP transfers. The host IP address displays in the dropdown list of options. Or in **Shell**, enter an IP address. For example, *192.0.2.1*. |
```